### PR TITLE
cleanup packages and migrated to package.xml 2 format

### PR DESCRIPTION
**Pull request by benmaidel**
this pr includes fixes to install tags and also the cob_arc android apk
- do not add the apk to the install tags because this app should not be delivered by debian pkgs

### CODE REVIEW

**Comment by floweisshardt** [comment]
fixes #7 
@ipa-fxm and @ipa-nhg please review

File: cob_android_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -61,7 +61,7 @@ catkin_package(
 #  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 #)
 
-install(DIRECTORY msg
+install(DIRECTORY srv
```
Comment (fmessmer) [inline cob_android_msgs/CMakeLists.txt]: we do not install any msg/srv/action folder....installing the generated message header is handled by catkin....

**Comment by fmessmer** [comment]
BTW, while we are on it, can we cleanup the package.xml and the CMakeLists.txt of all `cob_android` packages?
E.g. in https://github.com/ipa320/cob_android/blob/indigo_dev/cob_android_msgs/CMakeLists.txt there are a lot of comments that are copy-and-pasted from the `cob_script_server`...

**Comment by benmaidel** [comment]
i have removed the wrong install tags and cleaned up all packages. I think this PR should now be fine.

File: cob_android_msgs/CMakeLists.txt [hunk 2]
```diff
@@ -3,87 +3,11 @@ project(cob_android_msgs)
 
 find_package(catkin REQUIRED COMPONENTS message_generation std_msgs)
```
Comment (fmessmer) [inline cob_android_msgs/CMakeLists.txt]: Your srv actually does not depend on std_msgs - so you can remove it from package.xml and CMakeLists

File: cob_android_resource_server/CMakeLists.txt [hunk 3]
```diff
@@ -2,89 +2,14 @@ cmake_minimum_required(VERSION 2.8.3)
 project(cob_android_resource_server)
 
 find_package(catkin REQUIRED COMPONENTS rospy)
```
Comment (fmessmer) [inline cob_android_resource_server/CMakeLists.txt]: no build_depend to rospy

File: cob_android_script_server/CMakeLists.txt [hunk 4]
```diff
@@ -1,29 +1,10 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_android_script_server)
 
-
-
-
-find_package(catkin REQUIRED COMPONENTS rospy message_generation tf std_msgs cob_srvs trajectory_msgs geometry_msgs control_msgs cob_sound move_base_msgs)
+find_package(catkin REQUIRED COMPONENTS rospy message_generation std_msgs)
```
Comment (fmessmer) [inline cob_android_script_server/CMakeLists.txt]: no build_depend on rospy

File: cob_android_script_server/CMakeLists.txt [hunk 5]
```diff
@@ -33,61 +14,14 @@ generate_messages(
   DEPENDENCIES geometry_msgs std_msgs
```
Comment (fmessmer) [inline cob_android_script_server/CMakeLists.txt]: Your service does not depend on std_msgs

File: cob_android_script_server/package.xml [hunk 6]
```diff
@@ -6,54 +6,20 @@
...
-  <!-- <test_depend>trajectory_msgs</test_depend> -->
-  <!-- <test_depend>geometry_msgs</test_depend> -->
-  <!-- <test_depend>control_msgs</test_depend> -->
-  <!-- <test_depend>move_base_msgs</test_depend> -->
-  <!-- <test_depend>cob_sound</test_depend> -->
-  <!-- <test_depend>rostest</test_depend> -->
 
   <export>
-    <cpp cflags="-I${prefix}/msg/cpp"/>
   </export>
```
Comment (fmessmer) [inline cob_android_script_server/package.xml]: you can remove the whole `export` block

Comment (benmaidel) [inline cob_android_script_server/package.xml]: i know, but i like it more this way

**Comment by fmessmer** [comment]
are you migrating to package.xml format 2 in a separate PR? I think it's ok to do it with this one...

**Comment by benmaidel** [comment]
yes, format 2 will be a separate PR

**Comment by benmaidel** [comment]
ok finally, i hope this pr is now clean

File: cob_android_script_server/package.xml [hunk 7]
```diff
@@ -6,54 +6,14 @@
...
-  <!-- <test_depend>tf</test_depend> -->
-  <!-- <test_depend>std_msgs</test_depend> -->
-  <!-- <test_depend>cob_srvs</test_depend> -->
-  <!-- <test_depend>trajectory_msgs</test_depend> -->
-  <!-- <test_depend>geometry_msgs</test_depend> -->
-  <!-- <test_depend>control_msgs</test_depend> -->
-  <!-- <test_depend>move_base_msgs</test_depend> -->
-  <!-- <test_depend>cob_sound</test_depend> -->
-  <!-- <test_depend>rostest</test_depend> -->
+  <run_depend>cob_android_msgs</run_depend>
```
Comment (fmessmer) [inline cob_android_script_server/package.xml]: requires a run_depend to `cob_script_server`

**Comment by fmessmer** [comment]
Thanks for taking this effort...LGTM

**Comment by benmaidel** [comment]
i've tested the install tags and they work as expected. So this means the `catkin_install_python` makro seems to work fine

**Comment by NadiaHG** [comment]
Test on cob4-2: OK

**Comment by fmessmer** [comment]
LGTM

**Pull request merged**